src/components/tables/cells/stats/Header.tsx

```diff
@@ -37,32 +46,61 @@ const StatsHeader = ({ header, selectableTableStoreName }: Props) => {
         SelectableTableStore['query']['count']
     >(selectableTableStoreName, selectableTableStoreSelectors.query.count);
 
+    const [firstColHeader, secondColHeader] = useMemo(() => {
+        return [
+            intl.formatMessage(
+                {
+                    id: headerSuffix ? headerSuffix : 'data.written',
+                },
+                {
+                    type: intl.formatMessage({
+                        id: header ?? 'data.data',
+                    }),
+                }
+            ),
+            intl.formatMessage(
+                {
+                    id: headerSuffix ? headerSuffix : 'data.read',
+                },
+                {
+                    type: intl.formatMessage({
+                        id: header ?? 'data.docs',
+                    }),
+                }
+            ),
+        ];
+    }, [header, headerSuffix, intl]);
```

Comment:
Since we need to pass the second column header in to two different things I just made the first column translate the same way to keep things consistent.